busylight/api/busylight_api.py

```diff
@@ -73,6 +81,25 @@ def __init__(self):
             self.username = None
             self.password = None
 
+        # Get and save the CORS Access-Control-Allow-Origin header
+        logger.info("Set up CORS Access-Control-Allow-Origin header, if environment variable BUSYLIGHT_API_CORS_ORIGINS_LIST is set.")
+        self.origins = json_loads(environ.get("BUSYLIGHT_API_CORS_ORIGINS_LIST", None))
```

Comment:
This raises a `TypeError` since `json_loads` doesn't know how to deserialize a `None`.  Make the default value an empty list instead:

```
self.origins = json_loads(environ.get("BUSYLIGHT_API_CORS_ORIGINS_LIST", [])
```

Reply:
Ah, oops! Fixing

Reply:
After this change, the existing pytest suite runs without error.

Reply:
Good point - how do I run the test suite?

Reply:
I think I ninja-edited my comment on you.

Reply:
Fixed it.

Reply:
Ah yes, *another* omission on my part. There are a bunch of development dependencies that are not installed by default and you can ask `poetry` to install them:

```console
<venv> $ poetry install --dev
...
```

After that, you should have things like `pytest` and `black` available to you.

You can run `pytest` from the command line:
```console
<venv> $ pytest
...
```

Reply:
I would just run `pytest` from the main `busylight` dir (root dir of the git repository)?

Reply:
I run it from the root of the git repo, which will use the pytest suite in `tests/`.

Reply:
Excellent. Just ran it and "448 passed, 12 xfailed". At least no red failures! 😄

Reply:
The "xfailed" tests are eXpected to fail, usually it's stuff I don't know how to test correctly yet. Cool